include/ttmlir/Dialect/TT/IR/TTOpsTypes.td

```diff
@@ -61,6 +65,16 @@ def TT_CoreCoordAttr : TT_Attr<"CoreCoord", "core_coord"> {
   let assemblyFormat = "`(` $y `,` $x `)`";
 }
 
+def TT_TileSizeAttr : TT_Attr<"TileSize", "tile_size"> {
+  let summary = "TT tile_size attribute";
+  let description = [{
+    TT core_coord attribute containing a single physical core coordinate.
```

Comment:
We should update this description, TT tile_size attribute containing a supported Tensix tile shape.